images/images.go

```diff
@@ -28,13 +28,13 @@ func (i Image) HasURL() bool {
 	return i.URL != ""
 }
 
-type ImageStore interface {
+type ImageProvider interface {
```

Comment:
`images` is already in the package name, so the consumer must say `images.ImageProvider` which has some repetition.

WDYT about `images.Provider` instead?

Reply:
Oh I like that!